### ISSUE TITLE
v.fmt not working

### ISSUE THREAD

**Issue by esimov** [vlang/vscode-vlang#523]
I have installed Vlang together with the vscode plugin on my Windows environment, but it looks like that none of the supported commands are working. For example when I'm trying to run `v.fmt` by pressing <kbd>CTRL+SHIFT+P</kbd> I'm getting the following error:

<img src="https://user-images.githubusercontent.com/883386/221904517-e276634b-51a9-47e4-bbeb-50b73f5251a6.png">

**Comment by esimov** [comment]
I figured it out, that `v ls` wasn't installed and this was the reason that I was getting the above error. Maybe it would help the future developers who want to try out the language somewhere to be mentioned that `v ls` should be installed in order to use the VsCode plugin.

**Comment by JalonSolov** [comment]
The plugin used to ask if you wanted to install vls.  Perhaps this is a regression.

**Comment by esimov** [comment]
It didn't asked me.

**Comment by JalonSolov** [comment]
Did you enable vls in the options?

I don't use vscode, so not sure, but most editors have a way to set up "do this when saving a file" operations.

**Comment by esimov** [comment]
Yes, I did. Also `"editor.formatOnSave"` is set as true globally, but for `V` is not working unfortunately.

**Comment by robvanmieghem** [comment]
v.fmt is not called by vscode to format files on save. There is no explicit languageformatter registered for v, formatting is done through the language server. 
`"editor.formatOnSave"` does work for me though when vls is enabled.
